core/templates/components/button-directives/exploration-embed-button-modal.component.html

```diff
@@ -22,14 +22,14 @@ <h3>Embed this exploration</h3>
       To embed this exploration in another webpage, copy the following HTML into the source code of the webpage:
     </p>
 
-    <div class="oppia-embed-modal-code" ng-click="selectText($event)">&lt;iframe src="<[serverName]>/embed/exploration/<[explorationId]>" width="700" height="1000"&gt;</div>
+    <div class="oppia-embed-modal-code" (click)="selectText($event)">&lt;iframe src="{{serverName}}/embed/exploration/{{explorationId}}" width="700" height="1000"&gt;</div>
```

Comment:
When do we use `src` and when should we use `[src]`? Also, how are these both used? Could we stick to one?

Reply:
Here we can't use `[src]` as this is the actual text and not an HTML tag (notice the &lt and &gt in the code). In this case, only string interpolation will work.

Reply:
How do we know it is actual text and not an HTML tag?

Reply:
Because if it were an actual tag it would be rendered as an iframe but it is not the case here. In the modal, we see the text "<iframe src="...">..." and the not the iframe.